README.md

```diff
@@ -100,16 +103,37 @@ cd /srv/atd-airflow;
 # pull the changes
 git pull;
 
-# return to user-land
-exit;
+
 ```
 
+#### If there has been a change to the Airflow Docker image like when updating the Airflow verson:
+
 The production Airflow deployment uses a second Docker compose file which provides haproxy configuration overrides. To start the production docker compose stack use you must load both files in order:
 
 ```shell
+# dts-int-data-p01
+
+# become the superuser
+su -;
+
+# enter into the production airflow directory
+cd /srv/atd-airflow;
+
+# pull the fresh production image
+docker pull atddocker/atd-airflow:production
```

Comment:
This `docker pull` line does exactly what needs to be done, but what would you think of embracing the `compose` nature of the stack and doing a `docker compose pull;` instead? That way, should we do something wonky like rename the image or whatever, this documentation will stay evergreen and correct.

Reply:
making that change too - makes sense to pull the image that we've defined 💯